src/org-manager/store/reducers/organisation.reducer.spec.ts

```diff
@@ -28,31 +28,31 @@ describe('Organisation Reducer', () => {
     it('should return the initial state.activeOrganisations', () => {
       const action = new fromActions.LoadActiveOrganisation();
       const state = reducer(initialState, action);
-      expect(state.activeOrganisations).toEqual({orgEntities: {}, loaded: false, loading: true, searchString: ''});
+      expect(state.activeOrganisations).toBeDefined();
```

Comment:
query - tests seem to have deteriorated a bit here in terms of utility - is this because they are using real data now? Could we not check the state still somehow?